backend/main/models/community_users.go

```diff
@@ -99,6 +104,34 @@ func GetUsersForCommunityByType(db *s.Database, communityId, start, count int, u
 	return users, totalRecords, nil
 }
 
+func GetCommunityLeaderboard(db *s.Database, communityId, start, count int) ([]LeaderboardUser, int, error) {
+	var users = []LeaderboardUser{}
+	err := pgxscan.Select(db.Context, db.Conn, &users,
+		`
+		SELECT v.addr, count(*) AS score FROM votes v 
+		JOIN proposals p ON p.id = v.proposal_id
+		WHERE p.community_id = $1
+		GROUP BY v.addr
+		ORDER BY score DESC
+		LIMIT $2 OFFSET $3
+		`, communityId, count, start)
+
+	// If we get pgx.ErrNoRows, just return an empty array
+	// and obfuscate error
```

Comment:
can we remove these comments? I think they're getting duplicated around. The code here is self-explanatory.